guide/designing-products/personal-finance.md

```diff
@@ -113,14 +115,13 @@ For more on this use case, see the [daily spending case study]({{ '/guide/case-s
    layout = "float-left"
 %}
 
-Rent, utilities, subscriptions, car payments, grocery bills, and similar expenses need to be paid regularly on a weekly or monthly basis. Compared to day-to-day discretionary spending, these payments are higher in value, less frequent, and play a more important role in our lives. We can go without an extra coffee in the afternoon, but not paying rent or water bills is typically not an option.
+Higher value payments such as rent, utilities, car payments, and groceries are seldom paid for in cash. Instead, they are tied to service providers with ingrained payment mechanisms such as checks, debits, ACH, and wire transfers.
 
 </div>
 
-While grocery bills are still one-off payments, and we may visit different stores every week, many payments in this category are tied to specific service providers we have relationships with. Financial institutions also provide different mechanisms for recurring payments where businesses can regularly deduct money from accounts without customers having to confirm each one. Bitcoin transactions have to be initiated by the sender, so automated deductions would not be easily possible for non-custodial wallets. This could be solved with Lightning wallets that are always online and allow for more complex interactions between senders and recipients.
-
-The amounts involved in these transactions are higher and therefore require better security. While you can carry cash hand it to anyone directly, monthly expenses are typically done indirectly via transfers between both parties' banks. A manual payment might require you to log into your banks' website via username and password, along with a second security mechanism like [two-factor authentication](https://en.wikipedia.org/wiki/Help:Two-factor_authentication){:target="_blank"}.
+For recurring payments, some of these mechanisms let businesses regularly deduct money from accounts without customers approving each transaction.
 
+Bitcoin is different since the sender must initiate all transactions, meaning that automated deductions might not be possible—at least when the wallet is non-custodial. Lightning wallets, which are always online, could solve this by allowing more complex interactions between senders and recipients.
```

Comment:
```suggestion
Bitcoin is different since the sender must initiate all transactions, meaning that automated deductions might not be possible—at least when the wallet is non-custodial.
```

Reply:
Mentioning Lightning as a solution here, fleetingly, feels unnecessary.
We don't yet cover it in the guide and have no info on how to achieve this, yet.